core/roadmap.rst

```diff
@@ -0,0 +1,39 @@
+Roadmap
+-------
+
+I don't really think a strict roadmap is the way forward. They are quite
+cumbersome and suck the joy out of open-source. However, I do believe in having
+a broad and overarching plan detailing how things could play out.
+
+Obviously, these are quite volatile. They are subject to change frequently and
+you shouldn't rely on this at all. A better and more complete list would be the
+`coala issue tracker <https://github.com/coala/coala/issues>`_.
+
+Major Future Features
+=====================
+
+Aspects
```

Comment:
Holy shit, I didn't realize that. I'm terribly sorry (in my defence, the aspect team's application timeline happened entirely during the time my internet was out) - I'll mention that it's the aspect team's project.